src/test/kotlin/org/rust/ide/console/RsConsoleCompletionTest.kt

```diff
@@ -93,4 +93,34 @@ class RsConsoleCompletionTest : RsConsoleCompletionTestBase() {
         "is_empty",
         "index"
     ))
+
+    fun `test after statement`() = checkSingleCompletion("""
+        fn nop() {}
+        let frobnicate = 1;
+    """, """
+        nop();
+        let x = frobn/*caret*/
+    """, """
+        nop();
+        let x = frobnicate/*caret*/
+    """)
+
+    fun `test nested`() = checkContainsCompletion("""
+        fn nop() {}
+        let var1 = 1;
+        let var2 = 2;
+    """, """
+        nop();
+        fn foo() {
+            if true {
+                for i in 0..10 {
+                    let x = var/*caret*/
```

Comment:
I think Evcxr is correct here, because function has its own scope and [can't access variables from outer function](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=a72256e6bd3a4bbe97a3eb6f4388d165)